### DISCUSSION TITLE
Incorrect Wake Visuzlization

### DISCUSSION THREAD

**Discussion by Nellytado** [NatLabRockies/floris#905] [Q&A]
Hi, 

In theory, the lowest point of the wake should occur just after the tubrine. However the wake from the first turbine has a lowest point which is between the first and second? Is this a potential error? 

![image](https://github.com/NREL/floris/assets/165584201/97041c4c-dc39-4e78-990f-2fd70d1a4c18)
![image](https://github.com/NREL/floris/assets/165584201/f455bac0-937b-4548-89e3-6021bc22a0e0)

**Comment by rafmudaf** [comment]
@Nellytado I saw that this was opened and closed. If your question was resolved, please post the details for the benefit of the community of current and future users who see this discussion.

**Comment by Nellytado** [comment]
Sorry I thought I solved it but I did not :( Still unsure why this is happening.

**Reply by misi9170** [reply]
@Nellytado Is this with the Cumulative Curl model? This is an expected artifact of the Cumulative Curl (CC) model, I believe (@bayc , can you confirm?).

**Reply by Nellytado** [reply]
Yes this is with the cumulative curl model. Can you direct me to any papers that confirm this? I assumed it was simply because the wake was still expanding but once the second turbine occurs wakes superimpose and the greatest reduction in velocity is then seen after the second turbine.
